docs/getting-started/delete-on-dockstore.rst

```diff
@@ -0,0 +1,45 @@
+Deleting an Entry on Dockstore
+===================================
+
+Tutorial Goals
+--------------
+
+-  Deleting an entry
+-  Ensuring that the entry does not re-appear on Dockstore
+
+This tutorial walks through the process of deleting entries from Dockstore.
+
+Requirements for Deletion
+-------------------------
+
+An entry can only be deleted if it satisfies all the following:
+
+-  Registered via GitHub Apps,
+-  Has never been published,
+-  Was created on/after April 1, 2022, and
+-  Is not a checked or checker workflow.
+
+If your entry does not meet the requirements above, you can archive it using these steps.
```

Comment:
I'll link the documentation page for archiving in my next PR for https://ucsc-cgl.atlassian.net/browse/SEAB-5979